.editorconfig

```
@@ -18,16 +18,16 @@ trim_trailing_whitespace = true
 ############################### 
 # .NET Coding Conventions     # 
 ############################### 
-[*.cs] 
+
 # Organize usings 
 dotnet_separate_import_directive_groups = false
 dotnet_sort_system_directives_first = true
 
 # this. preferences 
-dotnet_style_qualification_for_field = false:error 
-dotnet_style_qualification_for_property = false:error 
-dotnet_style_qualification_for_method = false:error 
-dotnet_style_qualification_for_event = false:error
+dotnet_style_qualification_for_field = true:error 
+dotnet_style_qualification_for_property = true:error 
+dotnet_style_qualification_for_method = true:error 
+dotnet_style_qualification_for_event = true:error
```

```suggestion
dotnet_style_qualification_for_field = false:error
dotnet_style_qualification_for_property = false:error
dotnet_style_qualification_for_method = false:error
dotnet_style_qualification_for_event = false:error
```

I do not like the `this` qualifier, please leave these as is - it is a style choice that I prefer and a redundancy that I would not like to introduce.

---

Ok, generally I use the old style guide that was internal to MS a while back and used to be StyleCop oriented. Looks like things have changed a bit since I last updated that. If you use the underscore prefix then you don't need this on everything, but if you don't use the underscore it is making it clear if the variable is local or part of the class.